### PR TITLE
fix: id:52033 Add error styling to picker component

### PR DESCRIPTION
**Pull request by Farhan-emgage**
Add error styling to picker component

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/52033

File: src/styles/foundation/colors.scss [hunk 1]
```diff
@@ -103,7 +103,11 @@ $color-palette-data: (
     mid: #9e9e9e,
     danger: #d50000,
     link: #1976d2,
-  )
+  ),
+   error: (
```
Comment (weegrom) [inline src/styles/foundation/colors.scss:107]: No need for this

File: src/components/Picker/Picker.scss [hunk 2]
```diff
@@ -149,4 +149,19 @@ $stacking-order: (
...
+.error {
+  
+  .input::placeholder {
+    // This is the only place this color is used.
+    // stylelint-disable-next-line color-no-hex
+    color: #9c9798;
+  }
+
+  .backdrop {
+    background-color: color(error, light);
```
Comment (weegrom) [inline src/components/Picker/Picker.scss:163]: Add error styling in same manor as for other components. Ex: https://github.com/emgage/engage-ui/blob/master/src/components/TextField/TextField.scss#L59

Make certain to also include Delicious theme styling. Ex: https://github.com/emgage/engage-ui/blob/master/themes/Delicious/TextField.scss#L60

**Review by weegrom — CHANGES_REQUESTED** [review]
Please review comment and make needed adjustments.

File: src/components/Picker/Picker.scss (hunk 2 above)
Comment (Farhan-emgage) [reply src/components/Picker/Picker.scss]: @weegrom the changes is done

**Review by weegrom — APPROVED** [review]

**Pull request merged**